src/frontend/src/sass/_buttons.scss

```diff
@@ -48,14 +49,14 @@ a:hover {
 
 }
 
-bcgovbtn:focus,
-bcgovbtn:focus-visible {
+.bcgovbtn:focus,
+.bcgovbtn:focus-visible {
   outline: 4px auto -webkit-focus-ring-color;
 }
 
-.bcgovbtn--primary {
+.bcgovbtn__primary {
   background-color: $primary-colour;
-  color: $text-light;
+  --text-color: #{$text-light};
```

Comment:
wondering what adding `--` in front of `text-color` does?

Reply:
Oh! it is a CSS variable. We are trying to use both CSS custom properties and sass variables. It makes our css cleaner.
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties